lib/matplotlib/afm.py

```diff
@@ -246,55 +277,79 @@ def _parse_kern_pairs(fh):
     raise RuntimeError('Bad kern pairs parse')
 
 
+CompositePart = namedtuple('CompositePart', 'name, dx, dy')
+CompositePart.__doc__ = """
+    Represents the information on a composite element of a composite char."""
+CompositePart.name.__doc__ = """Name of the part, e.g. 'acute'."""
+CompositePart.dx.__doc__ = """x-displacement of the part from the origin."""
+CompositePart.dy.__doc__ = """y-displacement of the part from the origin."""
+
+
 def _parse_composites(fh):
     """
-    Return a composites dictionary.  Keys are the names of the
-    composites.  Values are a num parts list of composite information,
-    with each element being a (*name*, *dx*, *dy*) tuple.  Thus a
-    composites line reading:
+    Parse the given filehandle for composites information returns it as a dict.
+
+    It is assumed that the file cursor is on the line behind 'StartComposites'.
```

Comment:
Same question about 'behind'.

Reply:
Same answer. `parse_composites` interprets the contents of a composites block, but not the initial `StartComposites`, which is already consumed by the previous parser function.